app/src/views/AccountMyFormsDref/DownloadImportTemplateButton/utils.ts

```diff
@@ -0,0 +1,297 @@
+import {
+    isDefined,
+    isTruthyString,
+} from '@togglecorp/fujs';
+import xlsx, {
+    type Border,
+    type Row,
+    type Style,
+    type Workbook,
+    type Worksheet,
+} from 'exceljs';
+
+import ifrcLogoFile from '#assets/icons/ifrc-square.png';
+import {
+    COLOR_LIGHT_GREY,
+    COLOR_PRIMARY_BLUE,
+    COLOR_PRIMARY_RED,
+} from '#utils/constants';
+
+function hexToArgb(hexStr: string, alphaStr = 'ff') {
+    const hexWithoutHash = hexStr.substring(1);
+
+    return `${alphaStr}${hexWithoutHash}`;
+}
+
+export const headerRowStyle: Partial<Style> = {
+    font: {
+        name: 'Montserrat',
+        bold: true,
+    },
+    fill: {
+        type: 'pattern',
+        pattern: 'solid',
+        fgColor: { argb: hexToArgb(COLOR_PRIMARY_RED, '10') },
+    },
+    alignment: {
+        vertical: 'middle',
+        horizontal: 'center',
+    },
+};
+
+const headingStyle: Partial<Style> = {
+    font: {
+        name: 'Montserrat',
+        color: { argb: hexToArgb(COLOR_PRIMARY_BLUE) },
+    },
+    alignment: {
+        horizontal: 'left',
+        vertical: 'middle',
+    },
+};
+
+const defaultCellStyle: Partial<Style> = {
+    font: {
+        name: 'Poppins',
+    },
+    alignment: {
+        horizontal: 'left',
+        vertical: 'top',
+        wrapText: true,
+    },
+};
+
+const inputBorderStyle: Partial<Border> = {
+    style: 'dashed',
+    color: { argb: hexToArgb(COLOR_PRIMARY_BLUE) },
+};
+
+const inputCellStyle: Partial<Style> = {
+    fill: {
+        type: 'pattern',
+        pattern: 'solid',
+        fgColor: { argb: hexToArgb(COLOR_LIGHT_GREY, '10') },
+    },
+    border: {
+        top: inputBorderStyle,
+        left: inputBorderStyle,
+        right: inputBorderStyle,
+        bottom: inputBorderStyle,
+    },
+    alignment: {
+        vertical: 'top',
+        wrapText: true,
+    },
+};
+
+export async function buildCoverWorksheet(
```

Comment:
This is not a generic function and cannot be used for other import templates. Let's rename the function to match that.